common/src/main/resources/common-config.yml

```diff
@@ -44,6 +44,9 @@ telemetry:
       user: vmagent
       port: 8429
       max-disk-usage: "4GB"
+      retention-min-time: "5m"
+      retention-max-time: "4h"
```

Comment:
should this be larger? cc @lnardai

Reply:
These values come as the default from prometheus. Unfortunately I don't see an option in agent mode like `max-disk-usage`. I think 4h should enough if for some reason we would have issues sending these metrics to Thanos. I didn't want to risk consuming lot of disk with higher values.

https://github.com/prometheus/prometheus/blob/40c1efe8bc68f74bd857edd857e1fc442275b56e/tsdb/agent/db.go#L48-L52

```go
var (
	DefaultTruncateFrequency = 2 * time.Hour
	DefaultMinWALTime        = int64(5 * time.Minute / time.Millisecond)
	DefaultMaxWALTime        = int64(4 * time.Hour / time.Millisecond)
)
```